day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,25 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(new)
+     @toppings + new
```

Comment:
When I see the `+` operator, I think of string concatenation, or math with integers. But toppings is an array. You may try some research into methods that work with arrays, much like the hmwk from tasks 1!

Reply:
I went back through and tried using `@toppings << new` instead of `@toppings + new` and it worked!